src/diffusers/pipelines/deepfloyd_if/pipeline_if.py

```diff
@@ -335,9 +321,6 @@ def run_safety_checker(self, image, device, dtype):
             nsfw_detected = None
             watermark_detected = None
 
-            if hasattr(self, "unet_offload_hook") and self.unet_offload_hook is not None:
-                self.unet_offload_hook.offload()
```

Comment:
I don't think we need to manually offload anything anymore; I don't think `unry_offload_hook` on pipeline is ever used